cmd/argocd-repo-server/main.go

```diff
@@ -67,6 +69,8 @@ func newCommand() *cobra.Command {
 
 	command.Flags().StringVar(&logLevel, "loglevel", "info", "Set the logging level. One of: debug|info|warn|error")
 	command.Flags().Int64Var(&parallelismLimit, "parallelismlimit", 0, "Limit on number of concurrent manifests generate requests. Any value less the 1 means no limit.")
+	command.Flags().IntVar(&listenPort, "listen-port", common.DefaultPortRepoServer, "Listen on given port for incoming connections")
```

Comment:
Minor - but nice to list the defaults.

Reply:
Minor - would just `--port` be ok?

Reply:
Fixed parameter name, default already displayed by Cobra